ShapeCrawler.Tests/Properties/Resources.resx

```diff
@@ -238,4 +238,7 @@
   <data name="_040_video" type="System.Resources.ResXFileRef, System.Windows.Forms">
     <value>..\Resource\040_video.pptx;System.Byte[], mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </data>
+  <data name="_041_tableOnly" type="System.Resources.ResXFileRef, System.Windows.Forms">
+    <value>..\Resource\041_tableOnly.pptx;System.Byte[], mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
```

Comment:
Could you please rename `Resource\041_tableOnly.pptx` on  `Resource\tables\tables-case001.pptx`.

There is an idea of no longer using `Resource\{xxx}.pptx` pattern for the naming of test file. Would be better to have folder for each shape type. I believe the navigation is better in this way.

Reply:
done